features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/internal/metric/SessionEndedMetricDispatcher.kt

```diff
@@ -0,0 +1,66 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.rum.internal.metric
+
+import com.datadog.android.api.InternalLogger
+import com.datadog.android.rum.internal.domain.scope.RumSessionScope
+import com.datadog.android.rum.internal.domain.scope.RumViewManagerScope
+import com.datadog.android.rum.model.ViewEvent
+import java.util.concurrent.ConcurrentHashMap
+
+internal class SessionEndedMetricDispatcher(private val internalLogger: InternalLogger) :
+    SessionMetricDispatcher {
+
+    private val metricsBySessionIdMap: ConcurrentHashMap<String, SessionEndedMetric> = ConcurrentHashMap()
```

Comment:
minor: can drop `Map` suffix, naming of the format `xByY` already implies that it is association here. Also can drop explicit type declaration here.

```suggestion
    private val metricsBySessionId = ConcurrentHashMap<String, SessionEndedMetric>()
```

Reply:
updated